### PR TITLE
Issue115

### PR DESCRIPTION
**Pull request by SubhajitCode**
Fixes #

#### Describe the changes you have made in this pr -
Hi fixed the issue #115 
there was a possible problem in /public/js/logix.js file in Scope.prototype.fixLayout function 
it was increasing the height by 20 which i changed to 10 and it  fixed the bug #115 
@satu0king  please review it ... and let me know
thanking you 
Subhajit Barh

### Screenshots of the changes (If any) -
fixed

### CODE REVIEW

**Comment by SubhajitCode** [comment]
kindly ignore first two commits

**Comment by SubhajitCode** [comment]
OKK I changed the docker file it is now ready for merge

**Comment by armudgal** [comment]
Hi, please pull the code from our master for `circleci`, @satu0king  will take a look after that. Thanks

**Comment by SubhajitCode** [comment]
@armudgal what is circleci can you please provide the link ?? 
thanks subhajit barh

**Comment by armudgal** [comment]
@SubhajitCode CircleCI is a Continuos Integration tool which makes sure that any code change does not break anything. Since we included the config file later on you need to pull our master branch to update your branch. Easiest way to do this is `git pull https://github.com/CircuitVerse/CircuitVerse.git master`.

You can also see that there would be something mentioned as `All checks have failed` and you can go to `Details` to check out CircleCI

**Comment by SubhajitCode** [comment]
I made the necessary changes and all checks now successfully passed 
@armudgal @satu0king

File: public/js/logix.js [hunk 1]
```diff
@@ -244,7 +244,7 @@ Scope.prototype.fixLayout = function() {
     for (var i = 0; i < this.Output.length; i++)
         max_y = Math.max(this.Output[i].layoutProperties.y, max_y)
     if (max_y != this.layout.height)
-        this.layout.height = max_y + 20;
+        this.layout.height = max_y + 10;
 }
```
Comment (satu0king) [inline public/js/logix.js]: This looks correct. Will confirm and merge.

**Pull request merged**